doc/circuitikzmanual.tex

```diff
@@ -2438,6 +2447,135 @@ \subsubsection{Cute switches customization}
+\qquad
+\showanchors{rotary switch=25 in 120 wiper 10}{}(north/90/0.4, north east/45/0.4, east/0/0.4, south east/-45/0.4,
+    south/-90/0.4, south west/-135/0.4, west/180/0.4, north west/135/0.4, center/-120/0.3)
+
+In addition to the anchors they have in common with the cute switches, the rotary switch has the so called ``angled'' anchors and the ``external square anchors''. \emph{Angled anchors}, called \texttt{aout 1}, \texttt{aout 2} and so forth, are anchors placed on the output poles at the same angle as the imaginary lines coming from the input pole; \emph{square anchors}, called \texttt{sqout 1}\dots,  are located on an imaginary square surrounding the rotary switch on the same line.
+
+\bigskip
+
+\begin{circuitikz}
+    \begin{scope}[scale=2, transform shape]
+        \clip (-1,-.4) rectangle (1,.6);
+        \draw (0,0) node[rotary switch=9 in 90 wiper 10](Z){};
+        \draw [dashed, blue] (Z.cin) -- (Z.sqout 4);
+    \end{scope}
+    \path (Z.aout 4) \showcoord(aout 4)<-15:0.4>;
+    \path (Z.sqout 4) \showcoord(sqout 4)<0:0.4>;
+    \path (Z.out 3) \showcoord(out 3)<15:0.4>;
+    \path (Z.cout 3) \showcoord(cout 3)<180:0.4>;
+    \draw (8,0) node[rotary switch -=31 in 150 wiper 10](D){};
+    \foreach \i in {1,...,31} \draw (D.sqout \i) -- (D.aout \i);
+    \foreach \l/\a/\d in {north/90/0.2, north east/45/0.2, east/0/0.2,
+        south east/-45/0.2, south/-90/0.2, south west/-135/0.2, west/180/0.2, north west/135/0.2,
+    center/-145/0.7}
+    \path (D.ext \l) \showcoord(ext \l)<\a:\d>;
+    \path (D.aout 12)  \showcoordb(aout 12)<-5:0.4>;
+    \path (D.sqout 12)  \showcoordb(sqout 12)<0:0.4>;
+    \draw[blue, densely dotted] (D.ext north west) rectangle (D.ext south east);
+\end{circuitikz}
+
+The code for the diagram at the left, above, without the markings for the anchors, is:
+\begin{lstlisting}[basicstyle=\small\ttfamily]
+\begin{circuitikz}
+    \draw (8,0) node[rotary switch -=31 in 150 wiper 10](D){};
+    \foreach \i in {1,...,31} \draw (D.sqout \i) -- (D.aout \i);
+    \draw[blue, densely dotted] (D.ext north west) rectangle (D.ext south east);
+\end{circuitikz}
+\end{lstlisting}
+
+One possible application for the angled and the ``on square'' anchors is that you can use them to move radially from the output poles, for example for adding numbers:
+
+\begin{LTXexample}[varwidth=true, basicstyle=\small\ttfamily]
+\begin{circuitikz}
+\draw (0,0) node[rotary switch=13 in 120 wiper 0](S){};
+\foreach \i in {1,...,13} % requires "calc"
+    \path ($(S.aout \i)!1ex!(S.sqout \i)$)
+        node[font=\tiny\color{red}]{\i};
+\end{circuitikz}
+\end{LTXexample}
+
+Finally, notice that the value of width for the rotary switches is taken from the one for the ``cute switches'' which in turn is taken from the width of standard spdt, so that they match (notice that the ``center'' anchor is better centered in the rotary switch, so you have to explicitly align them).
```

Comment:
`spdt`?

Reply:
yes, the "traditional" single-pole double-throw switch.

Reply:
Makes sense, thanks!